jobsdb/jobsdb.go

```diff
@@ -521,10 +521,10 @@ const (
 // Loads db config and migration related config from config file
 func loadConfig() {
 	host = config.GetEnv("JOBS_DB_HOST", "localhost")
-	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
-	dbname = config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
+	user = config.GetEnv("JOBS_DB_USER", "rudder")
+	dbname = config.GetEnv("JOBS_DB_DB_NAME", "jobsdb")
 	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
-	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
+	password = config.GetEnv("JOBS_DB_PASSWORD", "password") // Reading secrets from
```

Comment:
Is it necessary to change these defaults?

Reply:
No, its not necessary.